Cargo.toml

```diff
@@ -53,7 +53,7 @@ license = "LGPL-3.0"
 homepage = "https://centrifuge.io/"
 repository = "https://github.com/centrifuge/centrifuge-chain"
 documentation = "https://reference.centrifuge.io/centrifuge_chain/index.html"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
Why a major version instead of 0.12.1?

Reply:
That's something I battled with internally myself. I thought this would be cleaner because v0.12.0 was related to enabling OpenGov on Altair.

Since this enables the metadata hash check, I think it deserves its major version. For similar releases in the future, I will take the minor version approach then since you seem to prefer that as well.